ansible/roles/cert-manager/templates/issuer.yml

```diff
@@ -29,9 +29,9 @@ spec:
     email: "{{ cert_manager.issuer_mail }}"
     # Name of a secret used to store the ACME account private key
     privateKeySecretRef:
-     name: letsencrypt-develop
+      name: letsencrypt-develop
```

Comment:
Ansible lint should not touch the file in templates folders because it can modify the indentation in kubernetes files and break the deployment

Reply:
@heroinedor The linting process won't make any changes, I just modified some indentation to match yaml standard formatting. I also successfully tested the whole ansible process, it shouldn't break anything 😉